### PR TITLE
Separate quick zoom gesture option

### PR DESCRIPTION
**Pull request by LukasPaczos**
Closes https://github.com/mapbox/mapbox-gl-native/issues/13759.

### CODE REVIEW

File: platform/android/MapboxGLAndroidSDKTestApp/src/main/java/com/mapbox/mapboxsdk/testapp/activity/camera/GestureDetectorActivity.java [hunk 1]
```diff
@@ -127,6 +124,11 @@ protected void onSaveInstanceState(Bundle outState) {
     mapView.onSaveInstanceState(outState);
   }
 
+  @Override
+  protected void onRestoreInstanceState(Bundle savedInstanceState) {
+    super.onRestoreInstanceState(savedInstanceState);
+  }
```
Comment (tobrun) [inline platform/android/MapboxGLAndroidSDKTestApp/src/main/java/com/mapbox/mapboxsdk/testapp/activity/camera/GestureDetectorActivity.java]: can be removed?

File: platform/android/MapboxGLAndroidSDKTestApp/src/main/java/com/mapbox/mapboxsdk/testapp/activity/camera/GestureDetectorActivity.java [hunk 2]
```diff
@@ -231,31 +236,50 @@ public boolean onCreateOptionsMenu(Menu menu) {
...
-  private void resetModes() {
-    focalPointLatLng = null;
-    mapboxMap.getUiSettings().setFocalPoint(null);
-    gesturesManager.getMoveGestureDetector().setMoveThreshold(0f);
-    mapboxMap.getUiSettings().setAllVelocityAnimationsEnabled(true);
-
-    if (marker != null) {
-      mapboxMap.removeMarker(marker);
-      marker = null;
+  private void focusOnAPoint(boolean focus) {
```
Comment (tobrun) [inline platform/android/MapboxGLAndroidSDKTestApp/src/main/java/com/mapbox/mapboxsdk/testapp/activity/camera/GestureDetectorActivity.java]: Not actionable but the naming of this method feels a bit strange to me

**Comment by LukasPaczos** [comment]
Yes, we'd like to expose this as well, good catch :+1:

**Comment by LukasPaczos** [comment]
8da4b5b05ae09971e0b3f895b050ef813d405edf adresses the comments, ready for another round.

**Pull request merged**